### PR TITLE
ServerResponseTask now has access to main activity context.

### PR DESCRIPTION
**Pull request by harshavardhana**
Accessing context allows FramesMap() to be accessed.
ServerResponseTask invokes ServerUpload() to finalize
the upload.

### CODE REVIEW

**Comment by harshavardhana** [comment]
Do not merge this just yet, frame association is not finished yet. :-1:

File: app/src/main/java/com/minio/io/alice/FrameHandler.java [hunk 1]
```diff
@@ -35,18 +35,15 @@
 public class FrameHandler implements Runnable {
     private LinkedList<byte[]> mQueue ;
```
Comment (deekoder) [inline app/src/main/java/com/minio/io/alice/FrameHandler.java]: Can't we simply make this public LinkedList instead of passing in context etc.

Comment (deekoder) [inline app/src/main/java/com/minio/io/alice/FrameHandler.java]: Or we can write a getter method if you want to keep it private.

Comment (harshavardhana) [inline app/src/main/java/com/minio/io/alice/FrameHandler.java]: We can do that this implementation needs to be changed to be able to safely accessible outside. Right now 'getFrame()' is the only synchronized way. 

But it is also causing delays in the code.. and android kills the threads not being able to process the server responses. 

You, me and @poornas need to discuss in detail.

File: app/src/main/java/com/minio/io/alice/ServerResponseTask.java [hunk 2]
```diff
@@ -20,7 +20,10 @@
 
 package com.minio.io.alice;
 
+import android.content.Context;
+import android.media.MediaActionSound;
```
Comment (poornas) [inline app/src/main/java/com/minio/io/alice/ServerResponseTask.java]: unneeded

Comment (harshavardhana) [inline app/src/main/java/com/minio/io/alice/ServerResponseTask.java]: done..

**Comment by harshavardhana** [comment]
Please review this @deekoder and @poornas

**Comment by deekoder** [comment]
@harshavardhana  does this works end to end.

File: app/src/main/java/com/minio/io/alice/XDebug.java [hunk 3]
```diff
@@ -25,5 +25,5 @@
 
 public class XDebug {
     // set this to true during development only.
-    public  static  final boolean LOG = true;
+    public static final boolean LOG = true;
```
Comment (poornas) [inline app/src/main/java/com/minio/io/alice/XDebug.java]: Should be turned off

**Pull request closed without merging**